crypto/modes/asm/aes-gcm-avx512.pl

```diff
@@ -200,14 +200,8 @@
 # ;;; Helper functions
 # ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-# ; Generates "random" local labels
-sub random_string() {
-  my @chars  = ('a' .. 'z', 'A' .. 'Z', '0' .. '9', '_');
-  my $length = 15;
-  my $str;
-  map { $str .= $chars[rand(33)] } 1 .. $length;
-  return $str;
-}
+# ; Counter variable that replaces previous usage of random_string() 
```

Comment:
Historical comments belong in the commit message, not the code comment ;-). This information is meaningless for the current implementation.
```suggestion
```

Reply:
deleted comment